apps/site/styles/locals.css

```diff
@@ -0,0 +1,5 @@
+html[lang='ko'] {
+  line-height: 1.8;
+  word-break: keep-all;
+  word-wrap: break-word;
```

Comment:
You have to use tailwind classes with `@apply` directive

Reply:
Thanks for the feedback! I'll make the changes and update the PR shortly😀